data-model/README.md

```diff
@@ -0,0 +1,29 @@
+
+This section includes the data model for different FAIR Digital Objects. 
+
+## Directory Structure
+
+- **annotations/**: Contains schemas and definitions related to annotation data model.
```

Comment:
I think this folder should no longer be there. They have become part of the fdo-type

Reply:
fixed